include/seqan3/std/concepts

```diff
@@ -466,7 +466,7 @@ concept bool StrictTotallyOrdered =
 
 /*!\interface   std::StrictTotallyOrderedWith
  * \extends     std::EqualityComparableWith
- * \brief       Requires std::Weakly_ordered bool and std::Equality_comparable.
+ * \brief       Requires weak order and std::EqualityComparable.
```

Comment:
why don't you use the seqan3::weakly_ordered_concept ?

Reply:
Yeah keep the link, otherwise it is tedious to understand what `weakly order` means

Reply:
Well, we have weak order in seqan3, but the stuff in `std` is completely self contained. I can change the description, but I don't want to refer to stuff in SeqAn.

Reply:
well he is not using the concept in the concept definition